src/pages/studyView/StudyViewUtils.tsx

```diff
@@ -1255,5 +1255,5 @@ export function submitToPage(url:string, params: { [id: string]: string }, targe
 }
 
 export function getClinicalEqualityFilterValuesByString(filterValues: string):string[] {
-    return filterValues.split(/(?<!\\),/).map(val=>val.trim().replace('\\,',','));
+    return filterValues.replace('\\,','$@$').split(",").map(val=>val.trim().replace('$@$',','));
```

Comment:
i'm pretty sure these replaces will only replace single instance, not global.   to make it globak, you need to use regexp with //g